frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -50,6 +50,12 @@ export const TextRow = ({
   const [isConfirmDeleteOpen, setIsConfirmDeleteOpen] = useState(false);
   const { t } = useTranslation();
 
+  useEffect(() => {
+    if (textId !== textIdValue) {
+      setTextIdValue(textId);
+    }
+  }, [textId, textIdValue]);
```

Comment:
Det virker ikke helt klart for meg hva denne `useEffect`en skal gjøre. Å kalle `setTextIdValue` i en `useEffect` som avhenger av `textIdValue` medfører også stor risiko for uendelig rekursjon. Finnes det andre måter å løse dette på?

Reply:
den trenger vi ikke , jeg har testet noe fra før og den kom på feil med merge conflikt , jeg fjernet den